CONTRIBUTING.md

```diff
@@ -35,6 +35,8 @@ To send us a pull request, please:
 3. Ensure local tests pass.
 4. Commit to your fork using clear commit messages.
 5. Send us a pull request, answering any default questions in the pull request interface.
+   1. Ensure you add the details on the change to the [CHANGELOG.md](CHANGELOG.md) file under `Unreleased` section.
+      If your change is a breaking change, label your change with `[breaking-change]` as prefix in `CHANGELOG.md`.
```

Comment:
So it's up to the PR creator to update the changelog file? If so, we may also want to include these reminders in the [PULL_REQUEST_TEMPLATE.md](https://github.com/partiql/partiql-lang-kotlin/blob/main/.github/PULL_REQUEST_TEMPLATE.md), since it could be easy to forget.

Reply:
Agreed, will add.